src/RazorSdk/Tasks/SdkRazorGenerate.cs

```diff
@@ -169,18 +169,18 @@ protected override string GenerateResponseFileCommands()
                     builder.AppendLine(RootNamespace);
                 }
 
-                if (!string.IsNullOrEmpty(CSharpLanguageVersion))
-                {
-                    builder.AppendLine("--csharp-language-version");
-                    builder.AppendLine(CSharpLanguageVersion);
-                }
-
                 if (GenerateDeclaration)
                 {
                     builder.AppendLine("--generate-declaration");
                 }
             }
 
+            if (!string.IsNullOrEmpty(CSharpLanguageVersion))
```

Comment:
We need to pass the CSharpLangVersion to Razor.Engine all the time. Otherwise it starts producing nullability code for 2.1 projects.